preview/css/_index.css

```diff
@@ -0,0 +1,61 @@
+/* Dependencies */
+@import "https://fonts.googleapis.com/css?family=Work+Sans:500,700&display=swap"; /* todo - add to eb-fonts */
+@import "normalize.css";
+@import "@ekstra-bladet/eb-colors/dist/eb-colors-classes.css";
+@import "@ekstra-bladet/eb-fonts/dist/eb-fonts-desktop.css";
+@import "_highlight";
+
+html,
+body {
+  margin: 0;
+
+  -webkit-font-smoothing: antialiased;
+}
+
+a {
+  color: #bd1118;
+  text-decoration: none;
+}
+
+div,
+section {
+  box-sizing: border-box;
+}
+
+h1,
+h2 {
+  line-height: normal;
+  margin: 0;
+}
```

Comment:
det her går ikke. det vil gøre at vi skal overskrive disse værdier i vores components også derfor at jeg længere nede har `.dsm-content > h1` og så videre for at sikre at det kun er det rundt om vores components der styles

Reply:
alle preview styling bør prefikses med dsm-SOMETHING

Reply:
Stylingen er tilføjet fra main.css for at components fremgår som de gør på eb.dk. Vi _skal_ overskrive ovenstående, hvis det skal fremgå anderledes.